frontend/packages/shipwright-plugin/integration-tests/features/shipwright-build-in-topology.feature

```diff
@@ -1,71 +1,85 @@
 @shipwright @odc-5387
 Feature: Shipwright build in topolgy
-              As a user, I want check my Shipwright Build in topology.
+    As a user, I want check my Shipwright Build in topology.
```

Comment:
We need to keep indentation in mind so that's why spaces are kept

Reply:
fixed the alignments